app/src/main/java/com/heig/atmanager/goals/GoalTodo.java

```diff
@@ -39,7 +39,8 @@ public void addQuantity(int quantity) {
     }
 
     public int getPercentage() {
-        return (int) (((float) quantityDone / goal.getQuantity()) * 100);
+        int percentage = (int) (((float) quantityDone / goal.getQuantity()) * 100);
```

Comment:
How can you be higher than 100% with Oo

Reply:
If you're a maniac and you did more squats than you should have